omod/src/test/resources/logEntriesTestDataset.xml

```diff
@@ -0,0 +1,8 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<dataset>
+    <idgen_log_entry id="1" source="Generator" generatedby="Admin" identifier="100892" comment="New patient " dateGenerated="3/10/2016"/>
+    <idgen_log_entry id="2" source="Generator" generatedby="Admin" identifier="100HH9" comment="Demo Data " dateGenerated="3/10/2017"/>
+    <idgen_log_entry id="3" source="Generator" generatedby="Admin" identifier="100H9T" comment=" " dateGenerated="1/10/2017"/>
```

Comment:
Did you intentionally do the above empty comment?

Reply:
It was intentional. I thought comments were ab optional field. Can modify though if it is supposed to be mandatory

Reply:
Are you aware of the fact that an empty string is different from null?

Reply:
Thats true. Going to fix that right away